server/controllers/popVerifyController.ts

```diff
@@ -0,0 +1,175 @@
+import { RequestHandler } from 'express'
+import path from 'path'
+
+export const renderSubmitPhoto: RequestHandler = async (req, res, next) => {
+  try {
+    res.render('pages/pop-id/submit-photo')
+  } catch (error) {
+    next(error)
+  }
+}
+
+export const renderOptions: RequestHandler = async (req, res, next) => {
+  try {
+    res.render('pages/pop-id/options', { selectedOption: req.session.idOption })
+  } catch (error) {
+    next(error)
+  }
+}
+
+export const handleOptions: RequestHandler = async (req, res, next) => {
+  try {
+    const { option } = req.body
+    req.session.idOption = option
+
+    const redirectPath = option === 'upload' ? '/pop/verify/upload-photo' : '/pop/verify/take-photo'
+    res.redirect(redirectPath)
+  } catch (error) {
+    next(error)
+  }
+}
+
+export const renderUploadPhoto: RequestHandler = async (req, res, next) => {
+  try {
+    const { errorMessage } = req.session
+    delete req.session.errorMessage
+    res.render('pages/pop-id/upload-photo', { errorMessage })
+  } catch (error) {
+    next(error)
+  }
+}
+
+export const handleUploadPhoto: RequestHandler = async (req, res, next) => {
```

Comment:
How do are we approaching wrapping this in the multipart form middleware handler?